baselines/webworker.generated.d.ts

```diff
@@ -2095,6 +2272,10 @@ type HeadersInit = Headers | string[][] | Record<string, string>;
 type BodyInit = Blob | BufferSource | FormData | URLSearchParams | ReadableStream | string;
 type RequestInfo = Request | string;
 type BlobPart = BufferSource | Blob | string;
+type HTMLOrSVGImageElement = never;
```

Comment:
do we need to write this? seems like noise at this point..

Reply:
CanvasImageSource uses this, we currently don't have a way to make it conditional. I will think about it.